docs/schemas/2024-04/donor-details.json

```diff
@@ -212,7 +212,16 @@
         }
       ],
       "type": "object",
-      "required": ["dateOfBirth", "email", "status"],
+      "required": ["dateOfBirth", "email", "status", "channel"],
```

Comment:
a breaking change on the shiny new schema, should this be `2024-04-1`?

Reply:
but also it isn't required, as the validation hasn't been updated yet. So maybe hold the `required` change for your next PR

Reply:
Good point!

Reply:
I think realistically we can break schema for now since no real LPAs are being registered, otherwise we're going to be making new versions pretty much daily.